src/test/java/step3/CarTest.java

```diff
@@ -0,0 +1,74 @@
+package step3;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static step3.Car.GO;
+import static step3.Car.STOP;
+
+class CarTest {
+
+    private Car car;
+
+    @BeforeEach
+    void setUp() {
+        car = new Car();
+    }
```

Comment:
```suggestion
    private final Car car = new Car();
```
junit의 기본 라이프사이클은 테스트메서드당이기 때문에 `@Beforeeach`로 초기화할 필요가 없습니다.

https://junit.org/junit5/docs/current/user-guide/#writing-tests-test-instance-lifecycle